src/components/App.jsx

```diff
@@ -2,13 +2,15 @@ import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
 import { hot } from 'react-hot-loader'
-import { translate } from 'cozy-ui/react/I18n'
-import { Sprite as IconSprite } from 'cozy-ui/react/Icon'
-import { Layout, Main } from 'cozy-ui/react/Layout'
+
+import 'cozy-ui/transpiled/react/stylesheet.css'
+import { IconSprite } from 'cozy-ui/transpiled/react'
+import { translate } from 'cozy-ui/transpiled/react/I18n'
+import { Layout, Main } from 'cozy-ui/transpiled/react/Layout'
 import { Route, Switch, Redirect, withRouter } from 'react-router-dom'
 
 import Sidebar from 'components/Sidebar'
-import Alerter from 'cozy-ui/react/Alerter'
+import Alerter from 'cozy-ui/transpiled/react/Alerter'
```

Comment:
Did you forgot to update this file and remove the "transpiled" in it ?

Reply:
Probably. We cannot trust Search and replace :(

Reply:
Thanks !